pkg/consoleplugin/contentserver/static/exposed-pendo-chunk-4c625175726bd3d902da.min.js

```diff
@@ -0,0 +1 @@
+"use strict";(self.webpackChunkconsole_plugin_template=self.webpackChunkconsole_plugin_template||[]).push([[484],{248:(t,n,e)=>{e.r(n),e.d(n,{default:()=>c}),function(t,n,e,i,o){var a,s,c,d,l;for((o=t[i]=t[i]||{})._q=o._q||[],s=0,c=(a=["initialize","identify","updateOptions","pageLoad","track"]).length;s<c;++s)!function(t){o[t]=o[t]||function(){o._q[t===a[0]?"unshift":"push"]([t].concat([].slice.call(arguments,0)))}}(a[s]);(d=n.createElement(e)).async=!0,d.src="https://{PENDO_HOST}/agent/static/{PENDO_KEY}/pendo.js",(l=n.getElementsByTagName(e)[0]).parentNode.insertBefore(d,l)}(window,document,"script","pendo");const i="toolchain.dev.openshift.com/",o=`${i}sso-user-id`,a=`${i}sso-account-id`;let s=!1;const c=(t,n)=>{if("identify"===t){const{user:t}=n;if(t){const n=t.metadata.annotations?.[o],e=t.metadata.annotations?.[a];n&&window.pendo&&(window.pendo[s?"identify":"initialize"]({visitor:{id:n},...e?{account:{id:e}}:null}),s=!0)}}}}}]);
```

Comment:
This doesn't look right. The content of the file is different. For example:
`webpackChunkconsole_plugin_template` vs `webpackChunktoolchain_member_web_console_plugin`. I remember fixing it and now you are reverting it back.